pom.xml

```diff
@@ -61,6 +61,41 @@
         <version>1.22</version>
         <optional>true</optional>
     </dependency>
+    <dependency>
+      <groupId>org.jenkins-ci.plugins.workflow</groupId>
+      <artifactId>workflow-step-api</artifactId>
+      <version>2.3</version>
+      <optional>true</optional>
+    </dependency>
+    <dependency>
+      <groupId>org.jenkins-ci.plugins.workflow</groupId>
+      <artifactId>workflow-support</artifactId>
+      <version>2.6</version>
```

Comment:
i double checked in the [files changed](https://github.com/jenkinsci/release-plugin/pull/22/files#diff-600376dffeb79835ede4a0b285078036R74) ,  workflow-support has the optional tag now

could you please merge this PR as I have fixed all the issues you pointed out? thanks in advance!